dev/Generated/TreeViewItem.properties.cpp

```diff
@@ -82,7 +82,7 @@ void TreeViewItemProperties::EnsureProperties()
                 winrt::name_of<double>(),
                 winrt::name_of<winrt::TreeViewItem>(),
                 false /* isAttached */,
-                ValueHelper<double>::BoxValueIfNecessary(12.0),
```

Comment:
How did this change?:)

Reply:
Looks like a code-gen'd change was not checked in ?